site/content/docs/getting-further/automation/target-scanning-issues.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Automation Guide - Target Scanning Issues"
+tags: 
+- automation
+- guide
+type: page
+---
+
+It is not unusual for target systems to struggle or even fail when being scanned by ZAP.
+
+This page explains what can go wrong, how to detect these problems and what can be done about them.
+
+### Cannot Connect
+
+If ZAP cannot connect to the target app then it will typically fail very quickly.
+Solving connection problems will depend on the underlying cause, which ZAP will not be able to detect.
+
+To confirm whether this is just a ZAP problem or something unrelated to ZAP then use another tool like [curl](https://curl.se/) to access the target from 
+exactly the same environment that ZAP is running in.
+
+#### Detection
+In this case detection is typically straightforward - exploring will fail.
+
+#### Target is Down
+This is nice and easy - start the target app!
+
+#### ZAP blocked by Firewall
+In this case the firewall needs to be configured to give ZAP access.
+
+This can be done via:
+* IP address range
+* A [Custom HTTP header](#custom-http-header)
```

Comment:
Mention this could be a host firewall on the system running zap, a host firewall on the target, or an infrastructure firewall between zap and the target??